SPECS/libguestfs/libguestfs.spec

```diff
@@ -1146,18 +1121,18 @@ rm ocaml/html/.gitignore
 %{_bindir}/virt-tar-out
 %{_mandir}/man1/virt-tar-out.1*
 
-%if %{with applicances}
+%if %{with appliances}
 %files tools
 %doc README
 %{_bindir}/virt-win-reg
 %{_mandir}/man1/virt-win-reg.1*
 %endif
 
 %files -n virt-dib
-%doc COPYING README
+%doc README
 %{_bindir}/virt-dib
 %{_mandir}/man1/virt-dib.1*
-%if %{with applicances}
+%if %{with appliances}
```

Comment:
Ah, well that's embarrassing on my part. Thanks for the spelling fix!

Reply:
No worries, I had no idea it was you. Had you said nothing, I would have blamed it on Fedora.;)